packages/react/src/components/status-task/StatusTask.tsx

```diff
@@ -1,10 +1,8 @@
 import React from 'react';
 import { TaskProp } from '../../shared/types/task';
 
-import './StatusTask.scss';
-
-export const StatusTask = ({ text }: TaskProp) => (
+export const StatusTask = ({ text, showText = true }: TaskProp) => (
```

Comment:
The StatusTask, PriorityTask, and ContactStatus components look very similar. Maybe extract some generic component?